browser/main/lib/dataApi/attachmentManagement.js

```diff
@@ -120,8 +120,13 @@ function fixLocalURLS (renderedHTML, storagePath) {
  * @param {Boolean} showPreview Indicator whether the generated markdown should show a preview of the image. Note that at the moment only previews for images are supported
  * @returns {String} Generated markdown code
  */
-function generateAttachmentMarkdown (fileName, path, showPreview) {
-  return `${showPreview ? '!' : ''}[${fileName}](${path})`
```

Comment:
why did you remove the parameter (and evaluation) ´showPreview´????

Reply:
I'm sorry, I'll fix it right away :smile: